### PR TITLE
Crewmate power roles

### PR DESCRIPTION
**Pull request by Ultradragon005**
- [x] Captain
- [x] Copycat
- [x] Dictator
- [x] Guardian
- [x] Lookout
- [x] Marshall
- [x] Mayor
- [x] Monarch
- [x] Overseer
- [x] President
- [x] Swapper
- [x] Telecommunications

### CODE REVIEW

File: Roles/Core/RoleBase.cs
Comment (Ultradragon005) [inline Roles/Core/RoleBase.cs:1]: temporary fix untill I fix roles which used another method

File: Patches/MeetingHudPatch.cs [hunk 1]
```diff
@@ -1210,24 +1207,16 @@ public static void Postfix(MeetingHud __instance)
 
                     } */
             //インポスター表示
+            sb.Append(seerRoleClass?.GetMark(seer, target));
```
Comment (Tommy-XL) [inline Patches/MeetingHudPatch.cs:1210]: Can you sync with the latest commit, since I already added `GetMark` support for modded players

**Comment by Ultradragon005** [comment]
amnesiac will be rewritten later..

**Comment by Ultradragon005** [comment]
I was doing some extra in another branch based from this branch, and found & fixed more problems, so some extra roles:
benefactor, chameleon, coroner

**Pull request merged**